lib/counter_culture/counter.rb

```diff
@@ -42,6 +44,10 @@ def change_counter_cache(obj, options)
                           else
                             counter_delta_magnitude_for(obj)
                           end
+
+        klass = relation_klass(relation, source: obj, was: options[:was])
+        primary_key = relation_primary_key(relation, source: obj, was: options[:was])
+
```

Comment:
Hmm yeah I think this was leftover from the original change. Can revert this part.